examples/tts/conf/uniglow.yaml

```diff
@@ -0,0 +1,100 @@
+name: &name "UniGlow"
+sample_rate: &sr 22050
+n_fft: &n_fft 1024
+n_mels: &n_mels 80
+fmax: &fmax 8000
+pad_value: &pad_value -11.52
+train_dataset: ???
+validation_datasets: ???
+resume_from_ckpt: null
+
+model:
+  sigma: 1.0
+  train_ds:
+    dataset:
+      cls: "nemo.collections.tts.data.datalayers.AudioDataset"
```

Comment:
Please change your yaml to comply with hydra's latest version.
IE use __target__ instead of cls and remove params